pkg/tfbridge/main.go

```diff
@@ -28,15 +29,33 @@ import (
 func Main(pkg string, version string, prov ProviderInfo, pulumiSchema []byte) {
 	// Look for a request to dump the provider info to stdout.
 	flags := flag.NewFlagSet("tf-provider-flags", flag.ContinueOnError)
+
+	// Discard print output by default; there might be flags such
+	// as -tracing that are unrecognized at this phase but will be
+	// parsed later by `Serve`. We do not want to print errors
+	// about them. Save `defaultOutput` for help below.
+	defaultOutput := flags.Output()
+	flags.SetOutput(ioutil.Discard)
+
 	dumpInfo := flags.Bool("get-provider-info", false, "dump provider info as JSON to stdout")
 	providerVersion := flags.Bool("version", false, "get built provider version")
-	contract.IgnoreError(flags.Parse(os.Args[1:]))
+
+	err := flags.Parse(os.Args[1:])
+	contract.IgnoreError(err)
```

Comment:
Is this still needed? It looks like the error is handled below.

Reply:
Yeah the only error that is handled is "--help" pseudo-error. Retaining for completeness... all this does is prints some info if debug level logging is asked  for.